script/update-json-schema.sh

```diff
@@ -1,5 +1,26 @@
 #!/bin/bash
 
-NEW_HASH=$(git ls-remote git://github.com/department-of-veterans-affairs/vets-json-schema/ refs/heads/master | awk '{print $1}')
-sed -i '' -e 's/\(vets-json-schema\.git#\)\(.*\)\(",$\)/\1'$NEW_HASH'\3/' package.json
+# Find the versions
+NEW_VERSION=$(git ls-remote --tags git://github.com/department-of-veterans-affairs/vets-json-schema/ | awk -F / '{ print $3 }' | sort -V -r | head -n 1)
+OLD_VERSION=$(grep -oP '(?<=department-of-veterans-affairs/vets-json-schema#)(.*)(?=",$)' package.json)
```

Comment:
the `vets-json-schema` entry is dangerously close to the end of the list of dependencies. Maybe to be extra super duper safe make that `,` before the end of the line optional?

Reply:
Ahh, yeah. Good call.